src/app/company-page/dialog/remove-dialog/remove-dialog.component.html

```diff
@@ -1,3 +1,5 @@
 <p>Are you sure you want to remove company {{data.name}}?</p>
+<div class="button-click">
 <button mat-raised-button (click) = "onCancel()">Cancel</button>
```

Comment:
Could you please arange the code here? The indentation is missing.

Reply:
Done